### PR TITLE
Fix paths in demos

### PR DESCRIPTION
**Pull request by mvorisek**
@ibelar please test

### CODE REVIEW

File: demos/atk-init.php [hunk 1]
```diff
@@ -121,28 +124,24 @@
...
+    // Send to demos source page in github.
     $ex = [
-        new \atk4\ui\jsExpression('const baseUrl = []', [$url]),
+        new \atk4\ui\jsExpression('const baseUrl = []', ['https://github.com/atk4/ui/blob/develop']),
         new \atk4\ui\jsExpression('window.open(baseUrl + document.location.pathname, "_blank")'),
     ];
 
-    // Send to demos source page in github.
     \atk4\ui\Button::addTo($layout->menu->addItem()->addClass('aligned right'), ['View Source', 'teal', 'icon' => 'github'])
         ->on('click', $ex);
```
Comment (mvorisek) [inline demos/atk-init.php]: @ibelar remove the `$img` below if it is not needed

File: demos/atk-init.php [hunk 2]
```diff
@@ -42,19 +42,21 @@
...
-// Need for phpUnit test only for producing right url.
+// Need for phpunit only for producing right url.
 $layout->name = 'atk_admin';
 $layout->id = $layout->name;
 
 if ($layout instanceof \atk4\ui\Layout\Navigable) {
-    $layout->addMenuItem(['Welcome to Agile Toolkit', 'icon' => 'gift'], ['/demos/index']);
+    $demosUrl = './demos/';
+
+    $layout->addMenuItem(['Welcome to Agile Toolkit', 'icon' => 'gift'], [$demosUrl]);
```
Comment (mvorisek) [inline demos/atk-init.php]: the original `index` was a typo?

Comment (mvorisek) [reply demos/atk-init.php]: I tested it now... It is horrible idea to append `.php` suffixes somewhere. User should either pass always a full path. Should be fixed.

Also reported at https://github.com/atk4/infrastructure/issues/13#issuecomment-621232048

File: demos/atk-init.php (hunk 1 above)
Comment (ibelar) [reply demos/atk-init.php]: fine. not sure why it was there in first place.

File: demos/atk-init.php [hunk 3]
```diff
@@ -42,19 +42,21 @@
...
 $layout = $app->layout;
-// Need for phpUnit test only for producing right url.
+// Need for phpunit only for producing right url.
 $layout->name = 'atk_admin';
 $layout->id = $layout->name;
 
 if ($layout instanceof \atk4\ui\Layout\Navigable) {
-    $layout->addMenuItem(['Welcome to Agile Toolkit', 'icon' => 'gift'], ['/demos/index']);
+    $demosUrl = preg_replace('~(?<=/demos/).+~s', '', $_SERVER['REQUEST_URI']);
+
```
Comment (ibelar) [inline demos/atk-init.php]: Why do you need 'REQUEST_URI' here? The value '/demos/' is only for setting proper link to menu item. There is no need fro preg_replace.

Comment (mvorisek) [reply demos/atk-init.php]: Consider having /demos/ in non web root and/or beiing in a demo in subdir.

Comment (mvorisek) [reply demos/atk-init.php]: @ibelar so ok to merge?

**Review by ibelar — APPROVED** [review]

**Comment by mvorisek** [comment]
@ibelar fixed also GH source link, check and merge if all ok

**Pull request merged**